### DISCUSSION TITLE
Getting Started with Wio Lite AI | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#597] [Q&A]
# Getting Started with Wio Lite AI | Seeed Studio Wiki

Wio Lite AI wiki

https://wiki.seeedstudio.com/Wio-Lite-AI-Getting-Started/

**Comment by hqbao** [comment]
Where can we can download the Template project (mentioned in STM32_AI_Integration.pdf) to start?
